README.md

```diff
@@ -29,7 +29,7 @@ the [How it works](docs/how-it-works.md) document is a good overview.
 
 ## Contributors ✨
 
-Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+Thanks goes to these wonderful people who have improved the code and documentation which helped in making this project grow. ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
```

Comment:
:art: suggestion to make this a bit more succinct

```suggestion
Thanks goes to these wonderful people who have improved the code and documentation to help this project grow. ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
```

Reply:
@shiftkey Better, thanks!!